CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## [0.30.0](https://github.com/eclipse-lemminx/lemminx/milestone/46?closed=1) (March 5, 2025)
+
+### Breaking Changes
+ * Update to Java 11. See [#1721](https://github.com/eclipse-lemminx/lemminx/pull/1721).
+    * _Rationale_: Several of lemminx's dependencies now require Java 11, so we were unable to update to the latest versions until we adopted Java 11.
```

Comment:
".. so we were unable to update to the latest versions until we adopted it."

Does that sound ok in terms of shortening ?